### PR TITLE
Document basic NUMA support

### PR DESCRIPTION
**Pull request by rmohr**
Documents https://github.com/kubevirt/kubevirt/pull/5846

### CODE REVIEW

File: docs/virtual_machines/numa.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
+# NUMA
+
+**FEATURE STATE:** KubeVirt v0.43
+
+## Predonditions
```
Comment (fabiand) [inline docs/virtual_machines/numa.md:5]: `Predonditions`, sorry, no idea!?

File: docs/virtual_machines/numa.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+In order to use current NUMA support, three preconditions must be met:
+[Dedicated CPU Resources](dedicated_cpu_resources.md) must be
+configured, [Hugepages](virtual_hardware.md#Hugepages) need to be allocatable on
+target nodes and the `NUMA`
+[feature gate](../operations/activating_feature_gates.md#how-to-activate-a-feature-gate)
+must be enabled.
```
Comment (fabiand) [inline docs/virtual_machines/numa.md:12]: WHat about using list items here?

File: docs/virtual_machines/numa.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
+# NUMA
+
```
Comment (fabiand) [inline docs/virtual_machines/numa.md:2]: Please add a small section with the scope of our "NUMA" support.

File: docs/virtual_machines/numa.md [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+target nodes and the `NUMA`
+[feature gate](../operations/activating_feature_gates.md#how-to-activate-a-feature-gate)
+must be enabled.
+
+## GuestMappingPassthrough
+
+GuestMappingPassthrough will pass through the node numa topology to the guest.
+The topology is based on the dedicated CPUs which the VMI got assigned from the
+kubelet via the CPU Manager. It can be requested by
+setting `spec.domain.cpu.guesstMappingPassthrough`.
```
Comment (fabiand) [inline docs/virtual_machines/numa.md:19]: VM? VMI? Splease note on which API
Typo: `gues*s*t.…`

File: docs/virtual_machines/numa.md [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+
+Since KubeVirt does not know upfront which exclusive CPUs the VMI will get from
+the kubelet, guests may see different NUMA topologies when being rescheduled.
+Further the resulting NUMA topology may be asymmetrical. Finally the VMI may
+fail to start on the node if not enough hugepages are available on the assigned
+NUMA nodes.
+
+While the NUMA modelling strategy has its limitations, for high-performance
+applications aligning the guest NUMA architecture with the node can be critical.
```
Comment (fabiand) [inline docs/virtual_machines/numa.md:28]: This sentence read's itself a little odd.

**Review by fabiand — COMMENTED** [review]
Some notes

File: docs/virtual_machines/numa.md (hunk 1 above)
Comment (cwilkers) [reply docs/virtual_machines/numa.md:5]:
```suggestion
## Preconditions
```

Also appropriate: Prerequisites.

File: docs/virtual_machines/numa.md (hunk 5 above)
Comment (cwilkers) [inline docs/virtual_machines/numa.md:28]:
```suggestion
While the NUMA modelling strategy has its limitations, aligning the guest's NUMA architecture with the node's can be critical for high-performance applications.
```

**Comment by fabiand** [comment]
ping?

File: docs/virtual_machines/numa.md (hunk 5 above)
Comment (rmohr) [reply docs/virtual_machines/numa.md]: Thanks. Took your sentence.

Comment (rmohr) [reply docs/virtual_machines/numa.md]: Fixed. :+1:

File: docs/virtual_machines/numa.md (hunk 4 above)
Comment (rmohr) [reply docs/virtual_machines/numa.md]: Done.

File: docs/virtual_machines/numa.md (hunk 3 above)
Comment (rmohr) [reply docs/virtual_machines/numa.md:2]: Added a section. Hope it is better now.

File: docs/virtual_machines/numa.md (hunk 2 above)
Comment (rmohr) [reply docs/virtual_machines/numa.md]: Done.

File: docs/virtual_machines/numa.md (hunk 1 above)
Comment (rmohr) [reply docs/virtual_machines/numa.md]: Fixed.

**Comment by rmohr** [comment]
Finally updated. PTAL.

**Pull request merged**